crates/net/network/src/session/active.rs

```diff
@@ -114,6 +114,8 @@ pub(crate) struct ActiveSession<N: NetworkPrimitives> {
     /// Used to reserve a slot to guarantee that the termination message is delivered
     pub(crate) terminate_message:
         Option<(PollSender<ActiveSessionMessage<N>>, ActiveSessionMessage<N>)>,
+    /// The range info for the peer.
+    pub(crate) range_info: Option<RangeInfo>,
```

Comment:
cool, this we must now handle here:

https://github.com/paradigmxyz/reth/blob/main/crates/net/network/src/session/active.rs#L265-L265

instead of emitting a broadcast we simply update the tracked range